packages/documentation-site/patternfly-docs/content/get-started/about-patternfly.md

```diff
@@ -0,0 +1,343 @@
+    </CardHeader>
+    <CardBody> Content guidelines that provide principles and best practices around writing for user experience. </CardBody>
+</Card>
+</GridItem>
+
+</Grid>
+
+<br />
+
+### Additional developer resources
+
+We offer more complex resources to assist with the development process.
+
+<Grid hasGutter>
+<GridItem span={4}>
+<Card isClickable isCompact isFullHeight>
+    <CardHeader
+        selectableActions={{
+            to: "/topology/getting-started",
+            selectableActionId: 'clickable-selectable-card-input-10',
+          }}
+    >
+    <CardTitle component="h4"> **Topology** </CardTitle>
+    </CardHeader>
+    <CardBody> A method for modeling the arrangement of elements in a network. </CardBody>
+</Card>
+</GridItem>
+
+<GridItem span={4}>
+<Card isClickable isCompact isFullHeight>
+    <CardHeader
+        selectableActions={{
+            to: "/developer-resources/global-css-variables",
+            selectableActionId: 'clickable-selectable-card-input-11',
+          }}
+    >
+    <CardTitle component="h4"> **CSS variable system** </CardTitle>
+    </CardHeader>
+    <CardBody> CSS variables to use within code, which are tied into our design token system. </CardBody>
+</Card>
+</GridItem>
+
+<GridItem span={4}>
+<Card isClickable isCompact isFullHeight>
+    <CardHeader
+        selectableActions={{
+            to: "/utility-classes/about-utility-classes",
+            selectableActionId: 'clickable-selectable-card-input-12',
+          }}
+    >
+    <CardTitle component="h4"> **Utility classes** </CardTitle>
+    </CardHeader>
+    <CardBody> Additional CSS classes that allow you to further customize and modify UI elements, like alignment, spacing, and shadows. </CardBody>
+</Card>
+</GridItem>
+
+</Grid>
+
+<br />
+
+## Join the community
+
+At the core of PatternFly is our global community of designers, developers, and other UX professionals with a passion for open source—in other words, our Flyers. Whether we're contributing to PatternFly or just staying up to date on new releases, we work together to make PatternFly a vibrant community of passionate people. Together, we celebrate creativity and foster a sense of teamwork and unity. 
+
+The PatternFly community is never finished growing, and we want to keep it that way, so reach out whenever—we're always open.
+
+  <Grid sm={12} md={4} hasGutter className="pf-v5-u-my-lg pf-v5-u-text-align-center ws-building-grid">
+    <Card isCompact> 
+      <CardTitle className="pf-v5-u-font-size-l"> Chat with us </CardTitle>
+      <CardBody> 
+        We have a Slack workspace, where you can ask us questions and share any feedback. Just like PatternFly, our Slack channels are open for all and we encourage you to join to connect with the team.
+      </CardBody>
+      <Divider />
+      <CardFooter>
+        [Join our Slack workspace](https://www.redhat.com/mailman/listinfo/patternfly)
+      </CardFooter>
+    </Card>
+    <Card isCompact>
```

Comment:
Do we have a mailing list solution yet? The link for this card just stays on this page.

Reply:
still waiting on that to be finalized :/ so I just left it hoping we'll have the new link soon